### PR TITLE
ADD Dataload-Manager Model

### PR DESCRIPTION
**Pull request by peng9808**
Add data loading module, dataset crd

### CODE REVIEW

File: cmd/dataload-init/main.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	dataSetName := pvc.Spec.VolumeName
+	dataLoadRequest := createDataLoadRequest(dataSetName, *subDir)
+	dsClient, err := datastorev1alpha1.NewForConfig(config)
+	watcher, err := watchCustomResource(dsClient, namespace, dataSetName)
+	if err != nil {
+		log.WithError(err).Error("Failed to start watching custom resource")
+		return
+	}
+	defer watcher.Stop()
+	// 开始计时
```
Comment (panpan0000) [inline cmd/dataload-init/main.go:76]: Comments in English is more suitable for global open source

File: cmd/dataload-init/main.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	start := time.Now()
+	if err := createCustomResource(dsClient, dataLoadRequest, namespace); err != nil {
+		log.WithError(err).Error("Failed to create custom resource")
+		return
+	}
+	fmt.Println("Created custom resource")
```
Comment (panpan0000) [inline cmd/dataload-init/main.go:82]: debug code ...

File: cmd/dataload-init/main.go (hunk 1 above)
Comment (peng9808) [reply cmd/dataload-init/main.go:76]: Thanks, it has been processed. [](https://github.com/hwameistor/datastore/pull/17)

File: cmd/dataload-init/main.go (hunk 2 above)
Comment (peng9808) [reply cmd/dataload-init/main.go:82]: Thanks, it has been processed. [](https://github.com/hwameistor/datastore/pull/17)

File: build/datamanager/Dockerfile.amd64 [hunk 3]
```diff
@@ -1,5 +1,6 @@
-FROM alpine:latest
+FROM centos:7
```
Comment (yankay) [inline build/datamanager/Dockerfile.amd64:1]: alpine is a better choice for base-image than centos :-)

**Pull request merged**